sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -78,3 +78,12 @@ export enum LayoutTemplates {
    */
   CenteredCardTab = "centered-card-tab",
 }
+
+/**
+ * Supporting user personal information details.
+ */
+export interface SupportingUserPersonalInfo {
```

Comment:
This will need to be converted to a DTO, as [per this comment](https://github.com/bcgov/SIMS/pull/4937/files#r2214023175).

Reply:
As per our conversation, DTO child objects should also be named wth the proper suffix.
Please rename this to `SupportingUserPersonalInfoAPIOutDTO`.